.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,8 +1,10 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-
+labels: ":construction: blocked: accessibility input needed, :construction: blocked: engineering input needed, :construction: blocked: functional input needed,
+:construction: blocked: overlapping issue, :construction: blocked: reporter input needed, :construction: blocked: ux input needed"
```

Comment:
I don't think that we need to include the `overlapping issue` label on all new issues

Reply:
Removed here:
https://github.com/cerner/carbon-graphs/pull/294/commits/2a8491e378ffbcaf3d6fbfc52b992a88c5492bc9